lib/reporters/mini.js

```diff
@@ -18,22 +23,45 @@ function MiniReporter() {
 module.exports = MiniReporter;
 
 MiniReporter.prototype.start = function () {
-	return '';
+	var self = this;
+	this.interval = setInterval(function () {
+		self.spinnerFrame = (self.spinnerFrame + 1) % self.spinnerFrames.length;
+		self.write(self.spinnerChar() + self.currentTest);
```

Comment:
It's also a bit weird that `self.spinnerChar() + self.currentTest` is duplicated in `MiniReporter#test` too.